packages/mds-policy-author/api.ts

```diff
@@ -264,6 +322,23 @@ function api(app: express.Express): express.Express {
     }
   })
 
+  app.get(pathsFor('/admin/geographies/:geography_id'), async (req, res) => {
```

Comment:
We also need a read all geographies (with metadata) endpoint.

Reply:
We've already discussed this in another channel, but for the sake of others reading this, I plan to implement these endpoints in a separate branch.